instrumentation/ratpack-1.4/library/build.gradle.kts

```
@@ -7,6 +7,7 @@ dependencies {
   library("io.ratpack:ratpack-core:1.4.0")
 
   testImplementation(project(":instrumentation:ratpack-1.4:testing"))
+  testImplementation("io.ratpack:ratpack-guice:1.4.0")
```

Shouldn't it be a `testLibrary` instead? (version gets updated to the latest one when `latestDepTest` is executed)
```suggestion
  testLibrary("io.ratpack:ratpack-guice:1.4.0")
```

---

Ok, I was not familiar with `testLibrary` - it is something custom of the repository?

---

Yeah, `library`/`testLibrary` are custom gradle configurations we use to facilitate testing against different versions of the instrumented lib.